### ISSUE TITLE
OpenBMC support for softoff breaks xCAT rpower command 

### ISSUE THREAD

**Issue by whowutwut** [xcat2/xcat-core#3422]
In recent OpenbBMC builds, the activation of softoff support for controlling the servers breaks xCAT rpower support.  

The following needs to be done, and tracking with this issue.  
1)  Change the rpower off to be "hard off" to the chassis instead of the host 
2)  rpower stat needs to be aware of the chassis power off 
```
}[root@fs3 mid05tor12cn04]# curl -c cjar -b cjar -k -H "Content-Type: application/json" -X GET https://$IP/xyz/openbmc_project/state/enumerate
{
  "data": {
    "/xyz/openbmc_project/state/bmc0": {
      "CurrentBMCState": "xyz.openbmc_project.State.BMC.BMCState.Ready",
      "RequestedBMCTransition": "xyz.openbmc_project.State.BMC.Transition.None"
    },
    "/xyz/openbmc_project/state/chassis0": {
      "CurrentPowerState": "xyz.openbmc_project.State.Chassis.PowerState.Off",
      "RequestedPowerTransition": "xyz.openbmc_project.State.Chassis.Transition.Off"
    },
    "/xyz/openbmc_project/state/host0": {
      "CurrentHostState": "xyz.openbmc_project.State.Host.HostState.Running",
      "RequestedHostTransition": "xyz.openbmc_project.State.Host.Transition.Reboot"
    }
  },
  "message": "200 OK",
  "status": "ok"
```
However rpower state is not looking at chassis state and returns "on" 
```
[root@fs3 mid05tor12cn04]# rpower mid05tor12cn04 state
mid05tor12cn04: on
```

3) Open up the `softoff` function for rpower and implement this

The time of softoff gives the Host 5 minutes to power off, before it sends the hard power off (integrated 6/25 via https://gerrit.openbmc-project.xyz/#/c/4895/) so depending on the firmware levels, it's either 5 minutes or 45 minutes. 

Asked some questions here: https://github.com/openbmc/openbmc/issues/1933

**Comment by whowutwut** [comment]
Issue: 
1.  Resolved by #3423 
2.  Dependent on: openbmc/openbmc#1933, Need owner
3.  Need Owner

**Comment by whowutwut** [comment]
Repoening, still want to use this issue to track the softoff function
